### ISSUE TITLE
[Bug]: Nextcloud tries to log me back in after removing account

### ISSUE THREAD

**Issue by spacehitchhiker42** [nextcloud/desktop#4913]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

Nextcloud brings up a dialog box saying that i've been logged out of my account, and starts a browser with the login/grant permissions page.

### Steps to reproduce

1. Open the main dialog from the tray
2. click your account
3. click the 3-dots
4. click remove account
5. Restart Computer

### Expected behavior

Don't try to log me back into an account that i want to get rid of.
wait until i click on the tray icon so that i can add an account manually.
[debug.zip](https://github.com/nextcloud/desktop/files/9523949/debug.zip)

### Which files are affected by this bug

No files, it's an account issue

### Operating system

Windows

### Which version of the operating system you are running.

Windows 10

### Package

Other

### Nextcloud Server version

24

### Nextcloud Desktop Client version

3.6.0

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 3.3.6 to 3.4.0)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [ ] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by dave-kennedy** [comment]
Is this a dupe of #4715?

**Comment by spacehitchhiker42** [comment]
Nope, but it might be related. the problem on #4713 is that it keeps forgetting his credentials when he restarts. 
My problem is that Accounts that i've removed from the desktop client keep reappearing and trying to log me back in.

**Comment by Tegenwind** [comment]
Can reproduce on 3.6.0 (macOS). Hovever, deleting the account through the tray menu again seems to get rid of it permanently.

**Comment by Snaptags** [comment]
I had to remove the account manually via nextcloud.cfg in ~/AppData/Roaming/Nextcloud :(

**Comment by cClaude** [comment]
Same issue on Linux:

* To fix using client from system package: you need to edit `~/.config/Nextcloud/nextcloud.cfg`
* To fix using client from flatpak package: you need to edit `~/.var/app/com.nextcloud.desktopclient.nextcloud/config/Nextcloud/nextcloud.cfg`

**Comment by Sieboldianus** [comment]
I have the same issue.

1. 2nd Account removed
2. PC Restart
3. 2nd Account is listed again

I can repeat steps 1-3 over and over.

Version: `3.10.0` (Windows)

**Comment by UpcraftLP** [comment]
`3.10.1` (Windows, x64) here, same problem.

A full PC restart is not required to reproduce the bug.
Simply fully closing NextCloud via the tray icon and then restarting the application causes it to re-add the account and try to log back in.

**Comment by Rello** [comment]
Hello,

the client was enhanced significantly since the opening of this ticket.
If you are still encountering issues, please get back to us

**Comment by mqatgit** [comment]
In fact, I do have this issue still. 

Linux Mint 22.2 (zara), Nextcloud client version 3.17.2.
Wanted to remove an account to an external provider (mia in NL), the confirmation popup comes up, I confirm, Nextcloud seems to restart automatically, and immediately wants to re-authenticate the account I just deleted. That is regardless of whether I was still actively signed in to the account or not. The problem survives reboots of the machine.

I have not yet tried to edit the nextcloud.cfg as advised above, because of the hint at the 'significant enhancements'. The account enumeration in the cfg file does look as if it might be sensitive to arbitrary deletions of lines. The account in questions is no. 0, of 4 accounts over all.

By the way, I did the same on a current Windows client the other day and had no problem!


PS: went for editing the cfg file after all, with a backup, and it worked. The client just ignores the lacking account identification digit, and REMOVES the lines that I had commented out with ';'...
